articles/what-are-fleet-policies.md

```diff
@@ -4,11 +4,19 @@ Fleet policies are a great way to quickly monitor your devices by asking yes or
 
 Read more about [getting and staying compliant across your fleet.](https://fleetdm.com/use-cases/get-and-stay-compliant-across-your-devices-with-fleet)
 
-## Modify and add policies instantly when standards harden or expand
+## Add policies
 
-Standards can change based on business needs, industry best practices, or for any number of reasons. Teams grow, software requirements change, and new vulnerabilities are discovered every day. [Fleet makes adding and modifying policies easy](https://fleetdm.com/docs/using-fleet/rest-api#add-policy). To reduce the barrier of entry with Fleet for our less-technical users, we’ve introduced our standard templates that allow users to choose from a library of pre-made policies. Of course, those handy with [osquery](https://osquery.io/) and SQL can still create custom policies to their heart’s content. Simply click “Add a policy” from the “Policies” page in Fleet or select an existing policy to edit its query.
+To reduce the barrier of entry with Fleet for our less-technical users, we’ve introduced our [standard templates](fleetdm.com/queries) that allow users to choose from a library of pre-made policies. Of course, those handy with [osquery](https://osquery.io/) and SQL can still create custom policies to their heart’s content: 
```

Comment:
I would remove "for our less-technical users" and the "Of course,"  Keeps this positive.

Reply:
Hey @spokanemac can you open a separate PR for modifying this article? This change is just meant to improve the "discoverability" of the existing policy documentation. Thanks!